lib/__tests__/getJSON-test.js

```diff
@@ -3,7 +3,7 @@
 'use strict'
 
 import ExecutionEnvironment from 'exenv'
-import getJSON from '../getJSON'
+import mockrequire from 'mockrequire'

```

Comment:
should this be necessary? I think mockrequire is global when running mocha tests.